### DISCUSSION TITLE
Failed to copy vendor/ConEmu.xml.default file to vendor/conemu-maximus5/ConEu.xml!这个问题怎么解决

### DISCUSSION THREAD

**Discussion by 2shdhs** [cmderdev/cmder#2904] [Q&A]
Failed to copy vendor/ConEmu.xml.default file to vendor/conemu-maximus5/ConEu.xml!怎么解决

**Comment by daxgames** [comment]
Make sure Cmder root folder is somewhere where where you have write access to.  It's in the `README.md` don't put it in the program files folder

**Comment by DRSDavidSoft** [comment]
If you want to put it under `Program Files`, you have to manually add the permission to your account or the `Users` group under **Properties** → **Security** tab.

Here are some information with additional guidance: https://superuser.com/a/1141616
